pkg/kopia/command/snapshot.go

```diff
@@ -44,7 +40,6 @@ func SnapshotCreate(cmdArgs SnapshotCreateCommandArgs) []string {
 	args := commonArgs(cmdArgs.CommandArgs, requireLogLevelInfo)
 	args = args.AppendLoggable(snapshotSubCommand, createSubCommand, cmdArgs.PathToBackup, jsonFlag)
 	args = args.AppendLoggableKV(parallelFlag, parallelismStr)
-	args = args.AppendLoggableKV(progressUpdateIntervalFlag, longUpdateInterval)
 	args = addTags(cmdArgs.Tags, args)
```

Comment:
Removed this because it was already happening below in the function thus the ```--progress-update-interval``` flag was getting set twice